lib/nadia.ex

```diff
@@ -330,6 +330,23 @@ defmodule Nadia do
   def get_file(file_id), do: request("getFile", file_id: file_id)
 
   @doc """
+  Use this method to get link for file for subsequent use.
+  This method is an extension of the `get_file` method.
+
+  *Attention: link will be valid for 1 hour.*
+
+  Args:
+  * `file_id` - File identifier to get link
+  """
+  @spec get_file_link(binary) :: binary | {:error, Error.t}
+  def get_file_link(file_id) do
+    with {:ok, f} <- request("getFile", file_id: file_id),
```

Comment:
so, what arguments the `get_file_link` function takes? like, `%Nadia.Model.File` struct?